SIT/smoke/05_reserved_keyword/setup.cql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS origin.smoke_order;
+CREATE TABLE origin.smoke_order (col1 text primary key, "order" text);
+INSERT INTO origin.smoke_order(col1, "order") VALUES ('c1','o1');
+INSERT INTO origin.smoke_order(col1, "order") VALUES ('c2','o2');
```

Comment:
#nitpick Should we also think about adding another insert statement/updating this insert statement which tests LWT/CAS - WDYT?
```suggestion
INSERT INTO origin.smoke_order(col1, "order") VALUES ('c2','o2') IF NOT EXISTS;
```